### PR TITLE
feat: adds audit report

### PR DESCRIPTION
**Pull request by NelsonGaldeman**

### CODE REVIEW

**Review by Diogomartf — APPROVED** [review]

File: packages/app/src/pages/index.tsx [hunk 1]
```diff
@@ -131,6 +131,23 @@ export default function IndexPage() {
...
               so you can wrap your XDAI and start stacking.
             </Accordion>
+            <Accordion title="Is Stackly audited?">
+              Yes, Stackly contracts have been audited. Here you can read the {' '}
+              <ExternalLink
+                target="_blank"
+                href="https://ipfs.io/ipfs/QmUmmFkKvktZ14iA3237WuDrzNuhi4BMb4MoYMJHeFFbey"
+              >
+                full audit report
+              </ExternalLink>{' '}
```
Comment (berteotti) [inline packages/app/src/pages/index.tsx]: I think we should have a full stop here.

File: packages/app/src/pages/index.tsx [hunk 2]
```diff
@@ -131,6 +131,23 @@ export default function IndexPage() {
...
+              <br/><br/>
+              There is an active Security Bug Bounty, you can find more details {' '}
+              <ExternalLink
+                target="_blank"
+                href="https://app.hats.finance/bug-bounties/swapr-0x5dfc5451a739b4d94551195329bd37bcf8e4bf4a/scope"
+              >
+                here
+              </ExternalLink>{' '}
```
Comment (berteotti) [inline packages/app/src/pages/index.tsx]: And here too.

File: packages/app/src/pages/index.tsx (hunk 1 above)
Comment (NelsonGaldeman) [reply packages/app/src/pages/index.tsx]: Done

File: packages/app/src/pages/index.tsx (hunk 2 above)
Comment (NelsonGaldeman) [reply packages/app/src/pages/index.tsx]: Done

**Review by berteotti — APPROVED** [review]
Looking good for me!

**Pull request merged**